tests/stock_dashboard_api/views/test_stock_views.py

```diff
@@ -51,55 +57,93 @@ def test_delete_by_id(self):
         delete.return_value = True
         stock_id = 1
         with app.test_client() as client:
-            response = client.delete('/stocks/{stock_id}'.format(stock_id=stock_id))
-            self.assertEqual(response.status, '200 OK')
+            response = client.delete(BASE_URL + '{stock_id}'.format(stock_id=stock_id))
+            self.assertEqual(response.status, STATUS_200)
```

Comment:
I think, there are some predefined response classes, have you tried to find them?